libctru/source/svc.s

```diff
@@ -421,16 +421,12 @@ SVC_BEGIN svcContinueDebugEvent
 	bx  lr
 
 SVC_BEGIN svcGetProcessList
-	push {r0, r1}
+	str r0, [sp, #-0x4]!
```

Comment:
again, commit this code separately - you can use -m twice to give a title & a summary,

i.e. git commit -m "fix svcGetProcessList" -m "Fixing the implementation of svcGetProcessList - one output parameter not two."